embedchain/embedder/openai.py

```diff
@@ -1,7 +1,7 @@
 import os
 from typing import Optional
 
-from chromadb.utils.embedding_functions import OpenAIEmbeddingFunction
+from langchain_community.embeddings import OpenAIEmbeddings
```

Comment:
Instead of using from langchain_community, can we directly use the OpenAI APIs here for generating embeddings?

Reply:
sure,thx for the suggestion.

Reply:
@deshraj thx for the suggestion, fixed it!!